utils/check-working-tree/CHANGELOG.md

```diff
@@ -3,6 +3,12 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## Unpublished
```

Comment:
Actually, this is automatically generated during `lerna publish --conventional-commits`. I appreciate the motivation, though!